sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/bucket.scala

```diff
@@ -35,7 +36,8 @@ import org.apache.spark.sql.types.StructType
 private[sql] case class BucketSpec(
     numBuckets: Int,
     bucketColumnNames: Seq[String],
-    sortColumnNames: Seq[String])
+    sortColumnNames: Seq[String],
```

Comment:
`BucketSpec` is used in other places. Because `sortDirections` is optional, I don't want to change current implementation a lot.